File: filedata/entity/java/advertising.java

package entity;
// 
public class Advertising  extends BaseDomain{
    public Advertising() {

    }
    
    //
    private int(255) unsigned Id;
        
    //
    private String CompanyStyle;
        
    //
    private int(255) AdvertisingType;
        
    //
    private String AdvertisingTitle;
        
    //
    private String AdvertisingDetail;
        
    //
    private String AdvertisingImg;
        
    //
    private int(255) unsigned JumpType;
        
    //
    private String AdvertisingJumpUrl;
        
    //
    private Date UpdateTime;
        
    //
    private int(255) unsigned Status;
        
    //
    private int(255) unsigned Orders;
        
    public int(255) unsigned getId() {
        return Id;
    }
    public void setId(int(255) unsigned Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public int(255) getAdvertisingtype() {
        return Advertisingtype;
    }
    public void setAdvertisingtype(int(255) Advertisingtype) {
        this.Advertisingtype = Advertisingtype;
    }
            
    public String getAdvertisingtitle() {
        return Advertisingtitle;
    }
    public void setAdvertisingtitle(String Advertisingtitle) {
        this.Advertisingtitle = Advertisingtitle;
    }
            
    public String getAdvertisingdetail() {
        return Advertisingdetail;
    }
    public void setAdvertisingdetail(String Advertisingdetail) {
        this.Advertisingdetail = Advertisingdetail;
    }
            
    public String getAdvertisingimg() {
        return Advertisingimg;
    }
    public void setAdvertisingimg(String Advertisingimg) {
        this.Advertisingimg = Advertisingimg;
    }
            
    public int(255) unsigned getJumptype() {
        return Jumptype;
    }
    public void setJumptype(int(255) unsigned Jumptype) {
        this.Jumptype = Jumptype;
    }
            
    public String getAdvertisingjumpurl() {
        return Advertisingjumpurl;
    }
    public void setAdvertisingjumpurl(String Advertisingjumpurl) {
        this.Advertisingjumpurl = Advertisingjumpurl;
    }
            
    public Date getUpdatetime() {
        return Updatetime;
    }
    public void setUpdatetime(Date Updatetime) {
        this.Updatetime = Updatetime;
    }
            
    public int(255) unsigned getStatus() {
        return Status;
    }
    public void setStatus(int(255) unsigned Status) {
        this.Status = Status;
    }
            
    public int(255) unsigned getOrders() {
        return Orders;
    }
    public void setOrders(int(255) unsigned Orders) {
        this.Orders = Orders;
    }
             
    @Override
    public String toString() {
        return "advertising[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",AdvertisingType=" + AdvertisingType + ",AdvertisingTitle=" + AdvertisingTitle + ",AdvertisingDetail=" + AdvertisingDetail + ",AdvertisingImg=" + AdvertisingImg + ",JumpType=" + JumpType + ",AdvertisingJumpUrl=" + AdvertisingJumpUrl + ",UpdateTime=" + UpdateTime + ",Status=" + Status + ",Orders=" + Orders + "]";
    }
        
    
}
        

File: filedata/entity/c#/Advertising.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("advertising")]
    public class Advertising
    {
        
        /// <summary>
        /// 广告ID
        /// </summary>
        [Column("Id")]
        public int Id { get; set; }
    
    
        /// <summary>
        /// 公司后缀
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 广告类型
        /// </summary>
        [Column("AdvertisingType")]
        public int AdvertisingType { get; set; }
    
    
        /// <summary>
        /// 广告标题
        /// </summary>
        [Column("AdvertisingTitle")]
        public string AdvertisingTitle { get; set; }
    
    
        /// <summary>
        /// 广告内容
        /// </summary>
        [Column("AdvertisingDetail")]
        public string AdvertisingDetail { get; set; }
    
    
        /// <summary>
        /// 广告图链接
        /// </summary>
        [Column("AdvertisingImg")]
        public string AdvertisingImg { get; set; }
    
    
        /// <summary>
        /// 跳转方式 1=网络URL路径   2=本地H5
        /// </summary>
        [Column("JumpType")]
        public int JumpType { get; set; }
    
    
        /// <summary>
        /// 广告跳转地址
        /// </summary>
        [Column("AdvertisingJumpUrl")]
        public string AdvertisingJumpUrl { get; set; }
    
    
        /// <summary>
        /// 最后更新时间
        /// </summary>
        [Column("UpdateTime")]
        public DateTime UpdateTime { get; set; }
    
    
        /// <summary>
        /// 状态  0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public int Status { get; set; }
    
    
        /// <summary>
        /// 排序
        /// </summary>
        [Column("Orders")]
        public int Orders { get; set; }
    
    
    }
}
